### PR TITLE
YALB-250 and YALB-238: Wiring 2

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
## This PR addresses the following tickets:
- [YALB-250: Wire Text with Media](https://yaleits.atlassian.net/browse/YALB-250)
- [YALB-238: Wire Accordion and Accordion Item](https://yaleits.atlassian.net/browse/YALB-238)

### Relates to the following PRs:
- https://github.com/yalesites-org/yalesites_profile/pull/15
- https://github.com/yalesites-org/component-library-twig/pull/71

### Description of work
- Adds JS to Drupal for main menu and header toggle
- Adds JS to Drupal for accordion
- Wires the Text with Image paragraph
- Wires the Accordion and Accordion Item paragraphs

## Todo:
- [x] Accordion Item has the Text Paragraph nested in it. I wasn't expecting that. The text paragraph has some styling that is causing issues inside an accordion.
- [x] Accordion is missing a heading field
- [ ] Need a way (preprocess?) to tell accordion items whether or not the parent accordion has a heading (for semantics)

### CODE REVIEW

File: templates/paragraphs/paragraph--accordion.html.twig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{{ attach_library('atomic/accordion') }}
+
+{% embed "@molecules/accordion/accordion.twig" with {
+  accordion__hading: content.field_heading,
+} %}
+  {% block accordion__items %}
+    {{ content.field_accordion_item }}
```
Comment (nJim) [inline templates/paragraphs/paragraph--accordion.html.twig:7]: This is valid and works. I asked a similar question in a different PR around this pattern. I'm questioning if it makes sense to pass an entire drupal field to the component library. This feels overly opinionated. I wonder if things should loop in the Drupal template and then wire to something with a more predictable shape. Just mulling it over.

File: templates/paragraphs/paragraph--accordion.html.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{{ attach_library('atomic/accordion') }}
+
+{% embed "@molecules/accordion/accordion.twig" with {
+  accordion__hading: content.field_heading,
```
Comment (nJim) [inline templates/paragraphs/paragraph--accordion.html.twig:4]: Type heading.

**Review by nJim — CHANGES_REQUESTED** [review]
Found a typo and asked a question. Everything looks great.

File: templates/paragraphs/paragraph--accordion.html.twig (hunk 1 above)
Comment (ModulesUnraveled) [reply templates/paragraphs/paragraph--accordion.html.twig:7]: Same as before. This is a standard pattern. The component library component supports nested components/multi-value fields like this with Twig blocks.

File: templates/paragraphs/paragraph--accordion.html.twig (hunk 2 above)
Comment (ModulesUnraveled) [reply templates/paragraphs/paragraph--accordion.html.twig]: Fixed.

**Review by nJim — APPROVED** [review]
Looks great. Easy one to approve. Let's check with Vincent before shipping it.

**Pull request merged**